src/integration_tests/trafficcontroller/dropsonde_end_to_end_test.go

```diff
@@ -76,10 +77,15 @@ var _ = Describe("TrafficController for dropsonde messages", func() {
 	})
 
 	Context("Firehose", func() {
+
+		var (
+			messages <-chan *events.Envelope
+			errors   <-chan error
```

Comment:
There are three other contexts that don't require them. We could add a Streaming/Firehose contexts into their own context but I think it's fine the way it is.